app/Api/V1/Controllers/ProblemController.php

```diff
@@ -32,4 +36,33 @@ public function store(
         $service->createOne($params);
         return $this->response->created();
     }
+
+    public function index(
+        ProblemTransformer $problemTransformer,
+        ReadProblemService $service,
+        Request $request
+    ): Response
+    {
+        $limit = $request->get(self::QUERY_LIMIT, 10);
```

Comment:
constant should use const static variable (`10`)
in `ProblemController` there should be a constant called `DEFAULT_LIMIT`
then `$request->get(self::QUERY_LIMIT, self::DEFAULT_LIMIT);`